docs/welcome/glossary.md

```diff
@@ -66,4 +66,8 @@ PR Velocity tracks how quickly Pull Requests (PRs) are being merged in by identi
 
 ## Repository Insights
 
-The Repository Insights feature is a one-stop solution for understanding the pulse of your open-source projects and contributions. This feature is designed to provide a comprehensive view of your project's health and your individual contributions.
\ No newline at end of file
+The Repository Insights feature is a one-stop solution for understanding the pulse of your open-source projects and contributions. This feature is designed to provide a comprehensive view of your project's health and your individual contributions.
+
+## YOLO Coders
+
+A “YOLO coder” is an individual owner, contributor, or maintainer who pushes changes directly to the main branch of a repository without creating a pull request or seeking any community feedback. This risky practice bypasses essential review and collaboration processes.
```

Comment:
Just if we want to use the term "default branch" instead of "main branch".

```suggestion
A “YOLO coder” is an individual owner, contributor, or maintainer who pushes changes directly to the default branch of a repository without creating a pull request or seeking any community feedback. This risky practice bypasses essential review and collaboration processes.
```